input/input.css

```diff
@@ -0,0 +1,102 @@
+@import "../resetcss.css";
+
+
+@media screen and (max-width: 768px) {
+
+
+}
+
+
+
+/* ------------------------------------------------------------------------------------------------------------ */
+.skin{
+    padding: 0.5rem 0.5rem 0 0.1rem;
```

Comment:
don't convert values less than 10 px to rems as that will further reduce the size on smaller screens as suggested by Vikram

Reply:
ohkay i ll see that ,thanx